.travis.yml

```diff
@@ -27,5 +27,5 @@ script:
 
 # Run `pod lib lint` if specified
 - if [ $POD_LINT == "YES" ]; then
-    pod lib lint;
+    pod lib lint --allow-warnings;
```

Comment:
we may want to add a comment here so we don't forget to remove this after we remove survey support.